itables/html/initialize_offline_datatable.html

```diff
@@ -1,16 +1,3 @@
-<script>
-  // Create a pending promise immediately (this script is not deferred).
-  window.dataTablePromise = new Promise((resolve) => {
-    window.__resolveDataTablePromise__ = resolve;
-  });
-</script>
-<script type="module">
-  // Module scripts are automatically deferred, so this code will not execute immediately.
-  // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts
-  if ($.prototype.DataTable === undefined) {
-      const dt = (await import("dt_src")).default;
-      dt(window.$);
-      // Resolve the pending promise above, indicating that DataTable is now ready to be used.
-      window.__resolveDataTablePromise__();
-  }
-</script>
+<style>dt_css</style>
+<script>jquery_src</script>
+<script>window._datatables_src_for_itables = "dt_src"</script>
```

Comment:
This works and looks much simpler to me than the previous promise. Now `window._datatables_src_for_itables` is set when `itables.init_notebook_mode` is called, and its value is the base64 encoding of the `datatables.net` library.

Then for every table in the HTML document we do
```js
const dt = (await import(window._datatables_src_for_itables)).default;
```
which might cause some redundancy (unless import has a cache?).

IMO that's worth the simplification, and in practice I don't notice any performance difference. I also have hopes that this can solve an intermittent issue in VS Code (#216 ).

@fwouts may I ask if you see any problem with repeating the import like this?

Reply:
I think this is likely cached and so would be fine! You could try adding a simple `console.log("Hello, World!")` statement in the base64-encoded `window. _datatables_src_for_itables` to make sure it's only logged once for multiple tables.